src/components/select/bl-select.css

```diff
@@ -0,0 +1,224 @@
+
+.select-wrapper {
+  width: 100%;
+  position: relative;
+
+  --padding-vertical: var(--bl-size-2xs);
+  --padding-horizontal: var(--bl-size-xs);
+  --background-color: #fff;
+  --border-color: var(--bl-color-border);
+  --border-focus-color: var(--bl-color-primary-hover);
+  --icon-color: var(--bl-color-content-tertiary);
+  --text-color: var(--bl-color-content-primary);
+  --label-color: var(--bl-color-content-secondary);
+  --placeholder-color: var(--bl-color-content-tertiary);
+  --height: var(--bl-size-2xl);
+  --menu-padding: 0 var(--bl-size-m);
+  --menu-margin-top: var(--bl-size-2xs);
+  --font-size: var(--bl-font-size-m);
+  --disabled-color: var(--bl-color-tertiary);
+  --menu-height: 250px;
+}
+
+:host([size='large']) .select-wrapper {
+  --height: var(--bl-size-3xl);
+  --padding-vertical: var(--bl-size-xs);
+  --padding-horizontal: var(--bl-size-m);
+}
+
+:host([size='small']) .select-wrapper {
+  --height: var(--bl-size-xl);
+  --padding-vertical: var(--bl-size-3xs);
+  --padding-horizontal: var(--bl-size-xs);
+  --font-size: var(--bl-font-size-s);
+}
+
+.placeholder {
+  color: var(--placeholder-color);
+}
+
+:host([disabled]) .placeholder {
+  --placeholder-color: var(--bl-color-content-passive);
+}
+
+.invalid {
+  --border-color: var(--bl-color-danger);
+  --border-focus-color: var(--bl-color-danger-hover);
+  --label-color: var(--bl-color-danger);
+}
+
+.select-input {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  cursor: pointer;
+  outline: none;
+  box-sizing: border-box;
+  height: var(--height);
+  border: solid 1px var(--border-color);
+  font: var(--bl-font-title-3-regular);
+  padding: 0 var(--padding-horizontal);
+  border-radius: var(--bl-border-radius-s);
+  color: var(--text-color);
+}
+
+.remove-all {
+  margin-right: 4px;
+  font-size: var(--bl-font-size-xs);
+}
+
+.remove-all::after {
+  content: '';
+  position: absolute;
+  left: 1rem;
+  bottom: -3px;
+  height: 1rem;
+  border-left: 1px solid var(--bl-color-border);
+}
+
+.dropdown-icon {
+  font-size: var(--bl-font-size-m);
+}
+
+.select-open .select-input {
+  border: solid 1px var(--border-focus-color);
+}
+
+:host([disabled]) {
+  cursor: not-allowed;
+}
+
+:host([disabled]) .select-input {
+  pointer-events: none;
+  background-color: var(--disabled-color);
+}
+
+.select-input .selected-options {
+  padding: 0;
+  margin: 0;
+  list-style: none;
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+}
+
+.selected-options li {
+  display: inline;
+  font-size: var(--font-size);
+  color: var(--text-color);
+}
+
+.selected-options li:not(:last-child)::after {
+  content: ', ';
+}
+
+:host([disabled]) .selected-options li
+{
+  color: var(--bl-color-content-passive);
+}
+
+.select-input .actions {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  gap: var(--bl-size-2xs);
+  margin-left: var(--bl-size-2xs);
+}
+
+.select-menu {
+  --left: 0;
+  --top: 0;
+
+  position: absolute;
```

Comment:
When we develop tooltip component, we had issues about absolute positioning if container of the element has a limited height with `overflow: hidden` then we decided to use ‘fixed’ positioning. Maybe we can consider same approach for the consistency.

Reply:
Yes, we have issues with absolute positioning, they will be resolved when we change the positioning to `fixed`. But when we move to fixed positioning, we will have other problems. For example, during scrolling, the dropdown menu will appear in places we don't want. How do we find a solution to this situation? I think it will be more difficult to apply the solution in tooltip to select component.